eks-role-autoscaler.tf

```diff
@@ -1,4 +1,4 @@
-module "eks_iam_role_autoscaler" {
+module "eks_iam_assumable_role_autoscaler_eks" {
```

Comment:
```suggestion
## TODO: Proceed to renaming
# module "eks_iam_assumable_role_autoscaler_eks" {
module "eks_iam_role_autoscaler" {
```

Renaming will destroy the actual mapping and create a new one.
WDYT about keeping this PR focused on eks-public (and defering the renaming when we'll rename the current cluster cik8s)